lionblog-eunjae/src/routes/PostCreatePage.jsx

```diff
@@ -1,7 +1,26 @@
 const PostCreatePage = () => {
+	const handlePostCreatePageSubmit = () => {
+    alert("게시글을 등록합니다.") // TODO: add api call for sign up
+  };
 	return (
-		<div>Post Create Page</div>
+		<div className="flex flex-col items-center w-1/2">
+	    <h3 className="font-bold text-2xl">게시글 작성</h3>
+	    <form className="form gap-2" onSubmit={handlePostCreatePageSubmit}>
+				<label required htmlFor="title" className="label">제목</label>
+        <input required type="text" className="input" placeholder="제목을 입력해주세요."/>
```

Comment:
id 나 input 의 value와 관련된 속성도 추가해주면 더 좋을 것 같아요!!
```suggestion
        <input
          type="text"
          placeholder="제목을 입력하세요"
          id="title"
          value={post.title}
          className="input"
          required
        />
```